func/server/server.go

```diff
@@ -109,7 +124,10 @@ func run() error {
 	klog.Infof("Listening on %s", address)
 
 	// Start the gRPC server
-	server := grpc.NewServer()
+	server := grpc.NewServer(
+		grpc.MaxRecvMsgSize(o.pod.MaxGrpcMessageSize),
```

Comment:
I'm not good at grpc, but is sending multi-MB messages a good thing to do?

Reply:
I don't know if it is good practice or not, probably sending it in chunks via a grpc stream may be preferred where it makes sense. But, in this case we already have the architecture and the interfaces between components (porch-server <-> function-runner and function-runner <-> wrapper-server) that this fix doesn't want to change, and technically it works. Redesigning  the architecture is an option, but this PR only meant to be a hotfix for a bug.